projects/knora-ui/src/lib/viewer/operations/display-edit/display-edit.component.ts

```diff
@@ -51,6 +51,10 @@ export class DisplayEditComponent implements OnInit {
 
   }
 
+  ngAfterViewInit(): void {
+    // console.log(this.displayValueComponent);
+  }
+
```

Comment:
Seems like this is a remnant from debugging. Can we remove it (as well as the import & export)?

Reply:
done in https://github.com/dasch-swiss/knora-ui-ng-lib/pull/26/commits/59066f2762b9a001a2eefd39e6155aac23f34bd0